lib/statsd.rb

```diff
@@ -328,17 +319,16 @@ def format_event(title, text, opts={})
 
   private
 
-  def escape_event_content(msg)
-    msg.gsub! "\n", "\\n"
+  def event_content_escaped(msg)
+    msg.gsub "\n", "\\n"
   end
 
-  def rm_pipes(msg)
```

Comment:
Same as `event_content_escaped`, I think we should keep method names with imperative verbs as a convention.